recipes/mmft/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -ex
+
+RUST_BACKTRACE=full
+
+if [ "$(uname)" == "Darwin" ]; then
+    export HOME=`pwd`
+    echo "HOME is $HOME"
+
+    # according to https://github.com/rust-lang/cargo/issues/2422#issuecomment-198458960 removing circle ci default configuration solves cargo trouble downloading crates
+    #git config --global --unset url.ssh://git@github.com.insteadOf
+fi
```

Comment:
This is required anymore. See some recent Rust packages

* https://bioconda.github.io/contributor/guidelines.html#rust
* https://github.com/bioconda/bioconda-recipes/blob/master/recipes/lrge/build.sh

Reply:
@martin-g beat me to it!